src/solvers/changeCobraSolverParams.m

```diff
@@ -56,9 +56,9 @@
 allowedQPparams = {'minNorm', 'printLevel'};
 allowedMILPparams = {'intTol','relMipGapTol','timeLimit','logFile','printLevel'};
 
-% Only LP is currently included
+% Only LP, QP and MILP are currently included

```

Comment:
But global variables are not propagated to a parallel pool neither, so it doesn't really make any difference for parallel work.